config/php-cs-fixer.php

```diff
@@ -1,8 +1,6 @@
 <?php
 
-if (PHP_SAPI !== 'cli') {
-    die('This script supports command line usage only. Please check your command.');
-}
```

Comment:
Agree we probably want `strict_types` and not the CLI test.  If and as long as the tool only supports CLI, surely that would be reported elsewhere.